src/main/java/com/eternitywall/ots/Timestamp.java

```diff
@@ -127,13 +118,14 @@ public void serialize(StreamSerializationContext ctx) {
                 sortedAttestations.get(i).serialize(ctx);
             }
         }
+
         if (this.ops.isEmpty()) {
             ctx.writeByte((byte) 0x00);
 
             if (!sortedAttestations.isEmpty()) {
                 sortedAttestations.get(sortedAttestations.size() - 1).serialize(ctx);
             }
-        } else if (!this.ops.isEmpty()) {
+        } else if (!this.ops.isEmpty()) {      // TODO: Isn't this always true? Fix later.
```

Comment:
a leaf of the markle three, with attestations, has always 0 ops

Reply:
I was thinking strictly controll-flow here. My understanding of the underlying data structures is shaky.
But the code is like this:
`if (x) {
   // Do things
} else if (!x) {
}`
It is always the case that it is true, UNLESS "// Do things" changes x in some way. I did a quick look at the code and came up with my comment.

So, what do we do now? I promised this PR to be ONLY comments, which is why I did no code changes. Should I remove the comment, or change the code?

Reply:
OK, I removed this TODO, I think I will remember this issue later.

Reply:
thanks for the explanation 👍